Dockerfile

```diff
@@ -5,11 +5,8 @@ ARG service_alias
 ARG work_dir=/github.com/aws/aws-controllers-k8s
 WORKDIR $work_dir
 # For building Go Module required
-ENV GOPROXY=https://proxy.golang.org,direct
-ENV GO111MODULE=on
-ENV GOARCH=amd64
-ENV GOOS=linux
-ENV CGO_ENABLED=0
+ENV GOPROXY=direct
```

Comment:
I'm curious, did you encounter any issue caused by the Go proxy ?
We made [this change a month ago](https://github.com/aws/aws-controllers-k8s/pull/314) because `gomodules.xyz` was down for more than 24h and was causing all the service controllers build to fail

Reply:
Ah, yes. I kept running into sink-hole issue while I running builds with VPN. I modified this during testing and forgot to revert it back. Thanks for catching :D